test/integration/tas/tas_test.go

```diff
@@ -839,3 +839,56 @@ var _ = ginkgo.Describe("Topology Aware Scheduling", ginkgo.Ordered, func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("Topology validations", func() {
+	ginkgo.When("Creating a Topology", func() {
+		ginkgo.DescribeTable("Validate Topology on creation", func(topology *kueuealpha.Topology, matcher gomega.OmegaMatcher) {
```

Comment:
```suggestion
		ginkgo.DescribeTable("Validate Topology on creation", func(topology *kueuealpha.Topology, matcher gomega.OmegaMatcherr) {
```

Could you fix it here too?

Reply:
Good call :)

related: https://github.com/kubernetes-sigs/kueue/pull/3660#discussion_r1860070171